src/library/Uno.Material/Styles/Controls/HyperlinkButton.xaml

```diff
@@ -0,0 +1,91 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+					xmlns:winui="using:Microsoft.UI.Xaml.Controls"
+					mc:Ignorable="d">
+
+	<ResourceDictionary.MergedDictionaries>
+		<ResourceDictionary Source="../Application/Colors.xaml" />
+	</ResourceDictionary.MergedDictionaries>
+
+	<x:Double x:Key="MaterialHyperlinkButtonHoverOpacity">0.74</x:Double>
+	<x:Double x:Key="MaterialHyperlinkButtonPressedOpacity">0.38</x:Double>
```

Comment:
There are existing opacity constant for .74 and .38 in color.xaml; We don't want to reuse them?

Reply:
Because the name don't match
At first it was because I wasn't using the MaterialHover & MaterialPress opacities but I think you have point.